Dependencies/napi/napi-jsi/include/napi/napi-inl.h

```diff
@@ -1222,7 +1222,7 @@ namespace details
   struct Function {
     static inline jsi::Value Callback(napi_env env, const jsi::Value& thisVal, const jsi::Value* args, size_t count, void* data, Callable cb) {
       CallbackInfo callbackInfo{env, thisVal, args, count, {}, data};
-      return {env->rt, cb(callbackInfo)};
+      return jsi::Value(env->rt, cb(std::move(callbackInfo)));
```

Comment:
Why is this necessary?

Reply:
I was getting compilation issues with the callback usage in the Console polyfill and this change seemed to satisfy the compiler errors-- the compiler doesn't yell anymore when I back it out so I'm just going to revert the change. Thanks for pointing it out!